### PR TITLE
mini-test adapt

### PR DESCRIPTION
**Pull request by lec-bit**
due to kmesh multiple dataplane work, we need to adapt kmesh min-test:
add a cni.conflist and adapt test script, add the test script process to the cgroup net_cls and tag with kmesh

fixes https://github.com/kmesh-net/kmesh/issues/56

### CODE REVIEW

File: test/testcases/kmesh/libs/common.sh [hunk 1]
```diff
@@ -12,6 +12,11 @@ function env_init()
     yum localinstall -y fortio-*$(arch).rpm
     cd $CURRENT_PATH
 
+    cd /sys/fs/cgroup/net_cls && mkdir -p kmesh_test && cd kmesh_test
```
Comment (nlgwcy) [inline test/testcases/kmesh/libs/common.sh:15]: cgroup path needs to be deleted after the test case execution is complete.

**Comment by lec-bit** [comment]
image

File: test/testcases/kmesh/libs/common.sh (hunk 1 above)
Comment (lec-bit) [reply test/testcases/kmesh/libs/common.sh:15]: OK, add 'cgdelete net_cls:/kmesh_test' in cleanup

**Comment by lec-bit** [comment]
https://github.com/kmesh-net/kmesh/issues/56

**Review by nlgwcy — APPROVED** [review]
ok

**Pull request merged**